defaults/main.yml

```diff
@@ -48,9 +49,52 @@ openhpc_enable:
 ohpc_slurm_services:
   control: slurmctld
   batch: slurmd
-ohpc_release_repos:
-  "7": "https://github.com/openhpc/ohpc/releases/download/v1.3.GA/ohpc-release-1.3-1.el7.x86_64.rpm" # ohpc v1.3 for Centos 7
-  "8": "http://repos.openhpc.community/OpenHPC/2/EL_8/x86_64/ohpc-release-2-1.el8.x86_64.rpm" # ohpc v2 for Rocky 8
+ohpc_repos:
+  "7":
+    - name: OpenHPC
+      file: OpenHPC.repo
+      description: "OpenHPC-1.3 - Base"
+      baseurl: "http://build.openhpc.community/OpenHPC:/1.3/CentOS_7"
+      gpgcheck: true
+      gpgkey: https://raw.githubusercontent.com/openhpc/ohpc/v1.3.5.GA/components/admin/ohpc-release/SOURCES/RPM-GPG-KEY-OpenHPC-1
+    - name: OpenHPC-updates
+      file: OpenHPC.repo
+      description: "OpenHPC-1.3 - Updates"
+      baseurl: "http://build.openhpc.community/OpenHPC:/1.3/updates/CentOS_7"
+      gpgcheck: true
+      gpgkey: https://raw.githubusercontent.com/openhpc/ohpc/v1.3.5.GA/components/admin/ohpc-release/SOURCES/RPM-GPG-KEY-OpenHPC-1
+  "8":
+    - name: OpenHPC
+      file: OpenHPC.repo
+      description: OpenHPC-2 - Base
+      baseurl: "http://repos.openhpc.community/OpenHPC/2/CentOS_8"
+      gpgcheck: true
+      gpgkey: https://raw.githubusercontent.com/openhpc/ohpc/v2.6.1.GA/components/admin/ohpc-release/SOURCES/RPM-GPG-KEY-OpenHPC-2
+    - name: OpenHPC-updates
+      file: OpenHPC.repo
+      description: OpenHPC-2 - Updates
+      baseurl: "http://repos.openhpc.community/OpenHPC/2/updates/CentOS_8"
+      gpgcheck: true
+      gpgkey: https://raw.githubusercontent.com/openhpc/ohpc/v2.6.1.GA/components/admin/ohpc-release/SOURCES/RPM-GPG-KEY-OpenHPC-2
+
+default_ohpc_extra_repos:
+  "7":
+    - name: epel
+      file: epel.repo
+      description: "Extra Packages for Enterprise Linux 7 - $basearch"
+      metalink: "https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch&infra=$infra&content=$contentdir"
+      gpgcheck: true
+      gpgkey: "https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7"
+  "8":
+    - name: epel
+      file: epel.repo
+      description: "Extra Packages for Enterprise Linux 8 - $basearch"
+      metalink: "https://mirrors.fedoraproject.org/metalink?repo=epel-8&arch=$basearch&infra=$infra&content=$contentdir"
+      gpgcheck: true
+      gpgkey: "https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-8"
+
+extra_ohpc_extra_repos: []
```

Comment:
That name seems a bit odd

Reply:
How about having

- `ohcp_repos` as the FINAL list of repos, made up of concatenating:

  - `ohpc_openhpc_repos` - (currently `ohpc_repos`)
  - `ohpc_default_extra_repos` - i.e. epel, (currently `default_ohpc_extra_repos`)
  - `ohcp_extra_repos` - defaulting to `[]`

Reply:
Yeah, good idea - have implemented this but made `ohpc_extra_repos` into `openhpc_extra_repos` as an exposed, configurable var.